### DISCUSSION TITLE
Website not displaying correctly

### DISCUSSION THREAD

**Discussion by Kevdevui** [community/community#23904] [New to GitHub]
<p>Im having this issue wheree my website that I have uploaded onto github, is not displaying correctly, what  am I doing wrong, because i can see it displaying well in live server, but not well on github pages?</p>
<p>anyone got any answers?</p>
<p>Files Link<br>
<a href="https://github.com/Kevdevui/good_life" class="onebox" target="_blank" rel="noopener nofollow ugc">https://github.com/Kevdevui/good_life</a></p>
<p>Website Link<br>
<a href="https://kevdevui.github.io/good_life/" class="onebox" target="_blank" rel="noopener nofollow ugc">https://kevdevui.github.io/good_life/</a></p>

**Comment by tajmone** [comment, marked as answer]
<p>All your links are broken, including the repository link.</p>
<p>It seems that you’ve changed the repository name:</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/Kevdevui/GoodLifeStyle_Blog">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/Kevdevui/GoodLifeStyle_Blog" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/11064292/181079923-7456c55a-c7e6-43a3-b2fb-214e6f26ef20.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/5/f55bed55b6984e89f93036b6d736a5ecef54d7d4_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/f/5/f55bed55b6984e89f93036b6d736a5ecef54d7d4_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/f/5/f55bed55b6984e89f93036b6d736a5ecef54d7d4.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/5/f55bed55b6984e89f93036b6d736a5ecef54d7d4_2_10x10.png"></div>

<h3><a href="https://github.com/Kevdevui/GoodLifeStyle_Blog" target="_blank" rel="noopener nofollow ugc">GitHub - Kevdevui/GoodLifeStyle_Blog: Link below to Project</a></h3>

  <p>Link below to Project . Contribute to Kevdevui/GoodLifeStyle_Blog development by creating an account on GitHub.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>Also, be aware that letter-casing matters on GitHub, both for repository names as well as for filenames. So ensure that in your website links you are using correct upper/lower case, as in repository name and its folders and files.</p>

**Comment by Kevdevui** [comment]
<p>Thank you, I’ll have a look</p>
